map2.js

```diff
@@ -31,3 +31,19 @@ var phoneBook = [
 // For example, the 0th element of the new array should be {Abe: "1-111-111-1111"}
 
 //Your code here
+const newBook= phoneBook.map(people => {
+  //set object key to a variable, adding the [0] has it log as a string rather than a portion of an array 
+  let person= Object.keys(people)[0];
+  //will print out each name(key) as a string 
+  console.log(person);
+  //accesses string associated with person (?)
+  let number= people[person];
+  console.log(number);
+  number= '1-'+ number;
+  return [person, number];
```

Comment:
nice work! being nitpicky here: the instructions say to return an element. It's totally valid to do `return {person: number}`

You did all the hard work really well though, so I'm not worried about this detail

Reply:
woops, just realized you hard-coded an area code of `1` to each number. 

here's a hint: just like you can access elements in an array using `someArray[0]`, you can access characters in a string using `someString[0]`. Use this to grab the first character in each phone number and add it together with the whole phone number using string concatenation or string interpolation. you're really close!